elixir/apps/api/lib/api/client/views/relay.ex

```diff
@@ -1,34 +1,37 @@
 defmodule API.Client.Views.Relay do
   alias Domain.Relays
 
-  def render_many(relays, expires_at) do
-    Enum.flat_map(relays, &render(&1, expires_at))
+  def render_many(relays, expires_at, stun_or_turn) do
+    Enum.flat_map(relays, &render(&1, expires_at, stun_or_turn))
   end
 
-  def render(%Relays.Relay{} = relay, expires_at) do
+  def render(%Relays.Relay{} = relay, expires_at, stun_or_turn) do
     [
-      maybe_render(relay, expires_at, relay.ipv4),
-      maybe_render(relay, expires_at, relay.ipv6)
+      maybe_render(relay, expires_at, relay.ipv4, stun_or_turn),
+      maybe_render(relay, expires_at, relay.ipv6, stun_or_turn)
     ]
     |> List.flatten()
   end
 
-  defp maybe_render(%Relays.Relay{}, _expires_at, nil), do: []
+  defp maybe_render(%Relays.Relay{}, _expires_at, nil, _stun_or_turn), do: []
```

Comment:
Let's leave a comment for people reading this:
```suggestion
  # STUN returns the reflective candidates to the peer and is used for hole-punching;
  # TURN is used to real actual traffic if hole-punching fails. It requires authentication.
  # WebRTC will automatically fail back to STUN if TURN fails, 
  # so there is no need to send both of them along with each other.
  defp maybe_render(%Relays.Relay{}, _expires_at, nil, _stun_or_turn), do: []
```

Reply:
👍